### PR TITLE
initial-commit/ navbar for all pages

### PR DESCRIPTION
**Pull request by sakshi006**
Created a basic file structure and added navbar to the pages. Added routing where required.

### CODE REVIEW

File: src/Components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <Link to="/">KairoShop</Link>
+      </h2>
+      <div className="input-container">
+        <i className="fas fa-search icon"></i>
+        <input
+          className="input-field"
+          type="text"
+          value={searchValue}
+          placeholder="Search for something..."
+          onChange={(e)=>setSearchValue(e.target.vale)}
```
Comment (dev-enforced) [inline src/Components/Navbar/Navbar.jsx]: Minor change which you can correct later with the parameter of setSearchValue i.e. value instead of vale

**Review by dev-enforced — APPROVED** [review]
The component looks well structured. Had some minor suggestions while going through your code so you can check it out:
1) In the links section you can create a new component for the badges that display number over the icons  
2) you can use [react-icons](https://react-icons.github.io/react-icons/) too if ESLint gives errors while using Fontawesome icons
Rest looks good including the CSS structuring. Do deploy it on Netlify where we can see the live preview or the images of the component

**Comment by sakshi006** [comment]
> The component looks well structured. Had some minor suggestions while going through your code so you can check it out:
> 
> 1. In the links section you can create a new component for the badges that display number over the icons
> 2. you can use [react-icons](https://react-icons.github.io/react-icons/) too if ESLint gives errors while using Fontawesome icons
>    Rest looks good including the CSS structuring. Do deploy it on Netlify where we can see the live preview or the images of the component

Will do. Thanks. :)

File: src/Components/Navbar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  }
+  .btn:hover{
+      box-shadow: 1px 1px 4px 2px var(--primary);
+  }
+  
+  
+  
+  .input-field{
+      padding: 0.6rem 1.8rem;
+      border-radius: 0.8rem;
```
Comment (Asuraking1n) [inline src/Components/Navbar/Navbar.css]: border radius should be in px,  bcz like we don't want it to change according to screen size, so its bad practice to give border radius in rem vikram sir told about this in one of his sessions.

**Review by Asuraking1n — APPROVED** [review]
Every thing else looks good to me

File: src/Components/Navbar/Navbar.css (hunk 2 above)
Comment (sakshi006) [reply src/Components/Navbar/Navbar.css]: got it, will change

**Pull request merged**